modules/ROOT/pages/asm-secret-group-creation-task.adoc

```diff
@@ -1,11 +1,11 @@
-= Create a Secret Group (Anypoint Platform)
-ifndef::env-site,env-github[]
-include::_attributes.adoc[]
-endif::[]
+= Create and Edit Secret Group (Anypoint Platform)
 
+Using Secrets Manager you can create a secrets group and add secret types to it.
```

Comment:
Using Secrets Manager you can create a secrets group, a logical grouping of secrets, and add secret types to it.

Reply:
Rewrote as:

Secrets Manager enables you to create and edit secrets groups. It also enables you to create new secret types and add them to a secrets group.